scripts/apps/search/directives/ItemList.js

```diff
@@ -1950,6 +1967,10 @@ export function ItemList(
                     }
                 });
 
+                if (listConfig.thinRows) {
+                    scope.$on('item:previewed', listComponent.setNarrowView);
```

Comment:
Yes, I have  made the required changes. Please let me know if it needs any corrections.